CHANGELOG.md

```diff
@@ -1,21 +1,71 @@
 x.x.x Release notes (yyyy-MM-dd)
 =============================================================
-NOTE: Sync protocol version 3: CANNOT SYNC WITH MONGODB REALM CLOUD.
+NOTE: Realm file format is likely to break and you CAN NOT revert back to the previous file format - DATA WILL BE LOST!!!!
+NOTE: Sync protocol version 4: CANNOT SYNC WITH MONGODB REALM CLOUD.
 NOTE: Bump file format version to 21. NO DOWNGRADE PATH IS AVAILABLE.
 
 ### Enhancements
-* Adding Mixed types. ([#3389](https://github.com/realm/realm-js/issues/3389))
+* None.
+
+### Fixed
+
+* Performance regression for some scenarios of writing/creating objects with a primary key. ([realm/realm-core#4522](https://github.com/realm/realm-core/issues/4522))
+* Observing a dictionary holding links to objects would crash. ([realm/realm-core#4711](https://github.com/realm/realm-core/issues/4711), since v11.0.0-beta.1)
+
+### Compatibility
```

Comment:
Would be worth mentioning Studio. (I know it's not released yet, but you would know what it will be called)